packages/frontend/src/stores/messagelist.ts

```diff
@@ -432,18 +429,16 @@ class MessageListStore extends Store<MessageListState> {
             )
             jumpToMessageId =
               this.state.jumpToMessageStack[jumpToMessageStackLength - 1]
-            message = await BackendRemote.rpc.getMessage(
-              accountId,
-              jumpToMessageId as number
-            )
-            chatId = message.chatId
+            chatId = (
+              await BackendRemote.rpc.getMessage(accountId, jumpToMessageId)
+            ).chatId
           } else {
             const items = this.state.messageListItems
               .map(m =>
                 m.kind === 'message' ? m.msg_id : C.DC_MSG_ID_LAST_SPECIAL
               )
               .filter(msgId => msgId !== C.DC_MSG_ID_LAST_SPECIAL)
```

Comment:
what is DC_MSG_ID_LAST_SPECIAL (how can I find out? - searched for it in core but found no definition)

Reply:
IMU it's similar to [`DC_CHAT_ID_LAST_SPECIAL`](https://github.com/deltachat/deltachat-core-rust/blob/75fe4e106a740f8e3205d0bb56ce676fb2cd2411/deltachat-ffi/deltachat.h#L3562)

> // larger chat IDs are "real" chats, their messages are "real" messages